ledger-tool/src/main.rs

```diff
@@ -381,7 +383,26 @@ fn output_account(
     println!("  rent_epoch: {}", account.rent_epoch());
     println!("  data_len: {}", account.data().len());
     if print_account_data {
-        println!("  data: '{}'", bs58::encode(account.data()).into_string());
+        if encoding == UiAccountEncoding::Base58 {
+            println!("  data: '{}'", bs58::encode(account.data()).into_string());
+            println!("  encoding: base58");
+        } else {
+            let account_data = UiAccount::encode(pubkey, account, encoding, None, None).data;
+            match account_data {
+                UiAccountData::Binary(data, data_encoding_str) => {
```

Comment:
I wrote `data_encoding_str` badly in my example. It's not a str at this point, it's a variant of the UiAccountEncoding enum

Reply:
removed the suffix